app/views/aliquots/_tagged_aliquot.html.erb

```diff
@@ -3,7 +3,7 @@
   class="aliquot <%= container.location %> <%= container.state %>"
   rel="<%= "details_#{id}" %>"
   data-pool="<%= container.respond_to?(:pool) ? container.pool_id : '' %>">
-  <% container.aliquots.each_with_index do |aliquot| %>
-    <%= content_tag(:span, aliquot.tag_index, class: 'tag') %>
+  <% container.aliquots.each do |aliquot| %>
+    <%= content_tag(:span, aliquot.tag_index, class: ['tag', "tag-#{aliquot.tag_index.to_s.length}"]) %>
   <% end %>
```

Comment:
Pop the length of the tag index into the class to allow us to scale it down differently.